### PR TITLE
Renamed Update_fcs_file ,Updated the Docstrings,Created a comment

### PR DESCRIPTION
**Pull request by MdSufiyan005**

### CODE REVIEW

**Comment by cflynn3** [comment]
Welcome @MdSufiyan005 and thank you for your first contribution! 
Unfortunately, your PR fails our automated checks related to code formatting as well as our unit tests. Please have a look and follow the instructions in our [Contributing Guidelines](https://github.com/bp/ResSimpy/blob/master/docs/CONTRIBUTING.rst) for information around how to set up your own Poetry environment, and run the tests and checks on your local machine. The changes required for them to pass look fairly straightforward, so please feel free to make those changes and re-submit.
I also have a couple of minor suggested changes that I will comment on the PR with.

File: ResSimpy/Nexus/DataModels/FcsFile.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def update_fcs_file(self, new_file_path: None | str = None, new_include_file_loc
             file. Otherwise saves it to a path either absolute or relative to the file path provided.
             write_out_all_files (bool): Defaults to False. If False writes out only changed files.
             If False writes out all files.
-            preserve_file_names (bool): Defaults to False. If True will derive names from the existing fcs_file.
```
Comment (cflynn3) [inline ResSimpy/Nexus/DataModels/FcsFile.py:269]: Please revert this back to fcs_file.

File: ResSimpy/Nexus/DataModels/FcsFile.py [hunk 2]
```diff
@@ -255,9 +255,9 @@ def get_keyword_mapping() -> dict[str, tuple[str, type]]:
         return_dict = dict(single_keywords, **multi_keywords)
         return return_dict
 
-    def update_fcs_file(self, new_file_path: None | str = None, new_include_file_location: None | str = None,
+    def update_model_files(self, new_file_path: None | str = None, new_include_file_location: None | str = None,
                         write_out_all_files: bool = False, preserve_file_names: bool = False) -> None:
-        """Updates or creates a new fcs_file with all the updated include files.
+        """ Updates all the fcs files as well as the modified files.
```
Comment (cflynn3) [inline ResSimpy/Nexus/DataModels/FcsFile.py:260]: Please change to "Updates all the modified files as well as the fcs file if a new file has been created"

**Comment by MdSufiyan005** [comment]
Screenshot 2023-09-12 213606
Hey, can you tell me what should I do?
It is showing " cannot import name 'update_model_files' from 'ResSimpy.Nexus.DataModels.FcsFile "

**Comment by cflynn3** [comment]
> Screenshot 2023-09-12 213606 Hey, can you Please tell me what should I do? It is showing " cannot import name 'update_model_files' from 'ResSimpy.Nexus.DataModels.FcsFile "

Hi there, I'm afraid I'm not sure what is going wrong here as it looks like your Poetry environment has been set up OK, but you seem to get an error when running the tests that I personally do not get when attempting the same steps. Can you please try and pull the latest master and try again? Please also check firewalls etc. If that still fails perhaps we should arrange a call to try and debug this?

**Comment by MdSufiyan005** [comment]
Sure i will try it then

**Comment by MdSufiyan005** [comment]
hey @cflynn3 I did what you wanted me to do, While doing a test run I got 5 errors I solved 3 of them, but I am not able to solve the last 2 errors.

Screenshot 2023-09-14 122240

**Comment by cflynn3** [comment]
> hey @cflynn3 I did what you wanted me to do, But While doing a test run I got 5 errors I solved 3 of them, but I am not able to solve the last 2 errors. Can you please guide me.
> 
> Screenshot 2023-09-14 122240

Hi @MdSufiyan005 these failures will be related to your recent changes where you renamed that method. You should update all references to the previous method (which was `update_fcs_file`) in the project to your new name `update_model_files`

**Comment by MdSufiyan005** [comment]
Thanks

**Comment by MdSufiyan005** [comment]
I am really confused as to how to solve this issue (The last two errors).

**Comment by MdSufiyan005** [comment]
I am really confused as to how to solve this issue (The last two errors).

**Comment by cflynn3** [comment]
> I am really confused as to how to solve this issue (The last two errors).

OK, which part do you need more explanation of? The errors occur because you have changed the name of the method, but those two parts of the code are still trying to call that code using the old name. You need to search for instances of the old name in the code, and update them to the new name, and those errors will go away :-).

**Comment by MdSufiyan005** [comment]
Ok thanks i will try again.

**Pull request closed without merging**